packages/reports/addon/templates/components/dropdown-date-picker.hbs

```diff
@@ -8,7 +8,7 @@
     {{navi-date-picker
       date=_selectedDate
       dateTimePeriod="day"
-      onUpdate=(action (mut _selectedDate))
```

Comment:
Note: You don't need to wrap `(set)` in an `action`, it returns an action, essentially 😄

Reply:
Thanks! I was curious about that